src/Symfony/Component/Security/Core/Authorization/DebugAccessDecisionManager.php

```diff
@@ -1,98 +1,15 @@
 <?php
 
-/*
- * This file is part of the Symfony package.
- *
- * (c) Fabien Potencier <fabien@symfony.com>
- *
- * For the full copyright and license information, please view the LICENSE
- * file that was distributed with this source code.
- */
 
 namespace Symfony\Component\Security\Core\Authorization;
 
-use Symfony\Component\Security\Core\Authentication\Token\TokenInterface;
-
 /**
- * Decorates the original AccessDecisionManager class to log information
- * about the security voters and the decisions made by them.
- *
- * @author Javier Eguiluz <javier.eguiluz@gmail.com>
+ * @author Alessandro Lai <alessandro.lai85@gmail.com>
```

Comment:
you should remove this line, there is no authorship to claim here: git will record your contribution